baselines/FedPM/FedPM/conf/base.yaml

```diff
@@ -0,0 +1,54 @@
+---
+# this is the config that will be loaded as default by main.py
+# Please follow the provided structure (this will ensuring all baseline follow
+# a similar configuration structure and hence be easy to customise)
+
+dataset:
+  name: 'mnist' # [cifar10, mnist]
+  split: 'iid' # {iid, non-iid}
+  classes_pc: 4
+  minibatch_size: 64 #32
+
+model:
+  name: 'LeNet' # [Conv8/6/4, LeNet, Mask, Conv6]
+  mode: 'dense' # [mask, dense]
+  local_epochs: 3
+  activation: 'relu'
+  agg_steps: 5
+
+strategy:
+  _target_: # points to your strategy (either custom or exiting in Flower)
+  fedpm:
+    local_epochs: 3
+    local_lr: 0.1
+
+  fedavg:
+    local_lr: 0.0003
+    server_lr: 0.1
+
+  dense:
+    local_epochs: 3
+
+  sign_sgd:
+    local_lr: 0.01
+    server_lr: 0.0002
+
+client:
+  local_epochs: 3
+  local_lr: 0.1
+
+simulation:
+  fraction_fit: 0.2
+  fraction_evaluate: 0
+  n_clients: 10
+  n_rounds: 200
+  verbose: 0
+  store_output: True
+  strategy: 'fedpm' # [fedpm, dense]
+
```

Comment:
Please define another element in your config to set the `device` used by the server (e.g. to do global evaluation). You might want to pass this when defining your strategy's `evaluate_fn`. Something like this:

```
server_device: cpu # or cuda
```